### DISCUSSION TITLE
vscode platformio

### DISCUSSION THREAD

**Discussion by wimgrymonprez** [forntoh/LcdMenu#73] [Q&A] [answered]
I wanted to switch to vscode for easier programming with markup. I noticed the menu.update() function gives an error in vscode. In arduino ide it works. I only installed the library after the last update so it is the latest version. 2.1.2 . I don't know it i have to do an extra step or anything, just wanted to let you know in case this is affection more people than only me.

**Comment by forntoh** [comment]
Please share the error, I use vscode and platformio too

**Comment by wimgrymonprez** [comment]
The error i'm getting when using menu.update() is this one:

src/main.cpp: In function 'void loop()':
src/main.cpp:300:12: error: 'class LcdMenu' has no member named 'update'
       menu.update();
            ^~~~~~
*** [.pio\build\az-delivery-devkit-v4\src\main.cpp.o] Error 1

Verzonden vanuit Outlook<http://aka.ms/weboutlook>
________________________________
Van: Thomas Forntoh ***@***.***>
Verzonden: dinsdag 31 januari 2023 9:00
Aan: forntoh/LcdMenu ***@***.***>
CC: Wim Grymonprez ***@***.***>; Author ***@***.***>
Onderwerp: Re: [forntoh/LcdMenu] vscode platformio (Discussion #73)


Please share the error, I use vscode and platformio too

**Comment by forntoh** [comment, marked as answer]
It should be a cache issue, delete the `.pio` directory and rebuild the project.

**Comment by wimgrymonprez** [comment]
Hello again. Sorry for the extremely late reply. But you are right. I removed the folder and rebuilt the project and it worked! :-) Thank you very much.
